### PR TITLE
[IMP] purchase: improve usability

### PR DESCRIPTION
**Pull request by kbapt**
Description of the issue/feature this PR addresses: Improve usability

task 2040826

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/uom/views/uom_uom_views.xml [hunk 1]
```diff
@@ -23,20 +23,26 @@
...
-                                 e.g: 1 * (this unit) = ratio * (reference unit)
-                             </p>
+                             <label for="factor"
+                                attrs="{'invisible':[('uom_type','!=','smaller')]}"/>
+                            <div attrs="{'invisible':[('uom_type','!=','smaller')]}" class="o_row">
+                                <field name="factor"
+                                    digits="[42,5]"
+                                    attrs="{'readonly':[('uom_type','=','bigger')]}"/>
+                                <span class="oe_grey">
+                                    e.g: 1 * (reference unit) = ratio * (this unit)
```
Comment (sbuhl) [inline addons/uom/views/uom_uom_views.xml]: can you remove the spaces to have a wider input: e.g: 1*(reference unit)=ratio*(this unit)

File: addons/uom/views/uom_uom_views.xml [hunk 2]
```diff
@@ -23,20 +23,26 @@
...
+                                </span>
+                            </div>
+                            <label for="factor_inv" 
+                                attrs="{'invisible':[('uom_type','!=','bigger')]}"/>                           
+                            <div attrs="{'invisible':[('uom_type','!=','bigger')]}" class="o_row">
+                                <field name="factor_inv"
+                                    digits="[42,5]"
+                                    attrs="{'readonly':[('uom_type','!=','bigger')]}"/>
+                                <span class="oe_grey">
+                                    e.g: 1 * (this unit) = ratio * (reference unit)
```
Comment (sbuhl) [inline addons/uom/views/uom_uom_views.xml]: can you remove the spaces to have a wider input: e.g: 1*(reference unit)=ratio*(this unit)

File: addons/purchase/views/purchase_views.xml [hunk 3]
```diff
@@ -347,9 +336,9 @@
                                 <group>
                                     <field name="user_id"/>
                                     <field name="invoice_status"/>
-                                    <field name="payment_term_id" attrs="{'readonly': ['|', ('invoice_status','=', 'invoiced'), ('state', '=', 'done')]}"/>
+                                    <field name="payment_term_id" attrs="{'readonly': ['|', ('invoice_status','=', 'invoiced'), ('state', '=', 'done')]}" options="{'no_create': True}"/>
                                     <field name="fiscal_position_id" options="{'no_create': True}" attrs="{'readonly': ['|', ('invoice_status','=', 'invoiced'), ('state', '=', 'done')]}"/>
-                                    <field name="date_approve" groups="base.group_no_one"/>
+                                    <field name="date_approve" invisible="1"/>
```
Comment (RomainLibert) [inline addons/purchase/views/purchase_views.xml]: Just remove this field, it is already present in the view see line 166

File: addons/stock/views/stock_move_views.xml [hunk 4]
```diff
@@ -514,31 +514,6 @@
             </field>
         </record>
 
-        <record id="action_receipt_picking_move" model="ir.actions.act_window">
```
Comment (RomainLibert) [inline addons/stock/views/stock_move_views.xml]: I don't think we should remove this view, I know it is currently only used in purchase_stock and you removed the menuitem using it, but I'd keep this view nonetheless

**Comment by kba-odoo** [comment]
@RomainLibert I've done the changes as suggested

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**